app/views/payment/show.html.erb

```diff
@@ -1,22 +1,44 @@
-Category <%= @category.name %>
-<%= @category.Totalamount %>
-
-
-<% @payments.each do |item| %>
-    <div>
-        <%= item.payment.name %>
-        <%= item.payment.amount %>
-    </div>    
-<% end %>
-
-<br /><br />
-
-<%= link_to 'New payment', add_new_payment_path(@category.id) %>
-
-<br /><br />    
-
-<%= link_to 'Back', root_path %>
-
+<div class="bar">    
+    <%= link_to root_path do %>  
+        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M10.5 19.5L3 12m0 0l7.5-7.5M3 12h18" />
+        </svg>
+    <% end %>
+    <h2>Transactions</h2>
+    <%= link_to  do %> 
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6">
+            <path fill-rule="evenodd" d="M10.5 3.75a6.75 6.75 0 100 13.5 6.75 6.75 0 000-13.5zM2.25 10.5a8.25 8.25 0 1114.59 5.28l4.69 4.69a.75.75 0 11-1.06 1.06l-4.69-4.69A8.25 8.25 0 012.25 10.5z" clip-rule="evenodd" />
+        </svg>
+    <% end %> 
+</div>
+
+<div class="fakeNab">
+        Most recent
+        <p>-</p>
+</div>
+
+<div class="total">
+    <p>Total amount</p>    
+    $<%= @category.Totalamount %>
+</div>
+
+
+<div class='category'>
+    <% @payments.each do |item| %>
+        <div class="transaction">
+            <div class="text">
+                <%= item.payment.name %>
+                <br />
+                <%= item.payment.created_at.to_date.strftime("%d %B %Y") %>    
+            </div>    
+            $<%= item.payment.amount %>
+        </div>    
+    <% end %>
+</div> 
```

Comment:
@Dachrono I kindly suggest that you add a color to the text that is a little darker or that you add a light background color (like white) to the transaction item in order to provide the user with a more comfortable reading. I also suggest adding a little left and right margin to the transaction items, to separate it a little from the edge of the page.

Reply:
Tky for your feedback I really appreciate it, Solved